mne/realtime/lsl_client.py

```diff
@@ -83,6 +83,8 @@ def iter_raw_buffers(self):
             yield np.vstack(samples).T
 
     def _connect(self):
+        """To use this function with an LSL stream which has a 'name' but no 
+        'source_id', change the keyword in pylsl.resolve_byprop accordingly."""
```

Comment:
I'm not sure if that would be a good idea for the user to change this private function. 

one potential option is that we could pass both `prop` and `value` to the `LSLClient` instead of only `source_id` and it would be up to the user to make sure that it is the unique machine that met these criteria. we wouldn't want to have to resolve multiple items with the client.

Reply:
Just now saw your comment @teonbrooks, feel free to push a quick PR to clarify if the comment is not clear. I actually didn't read it as suggesting users change this function, more as a hint to us as devs in case we hit a problem/use case in the future we should change things here to fix it